tensorflow/tools/ci_build/builds/configured

```diff
@@ -37,4 +37,18 @@ fi
 SCRIPT_DIR=$( cd ${0%/*} && pwd -P )
 ${SCRIPT_DIR}/print_build_info.sh ${CONTAINER_TYPE} ${COMMAND[@]}
 
-${COMMAND[@]}
+# Execute the individual commands, separated with &&
+COMMAND+=("&&")
```

Comment:
Let's not do this. Those script are getting super complicated every day.

What if this whole feature was one or two lines in ci_parameterized_build.sh... just add "bash -c" to the command and you do not have to touch this script.

```bash
bash -c 'bazel build ... && bazel test -j 1 ...'
```